src/_guides/language/extension-methods.md

```diff
@@ -212,14 +213,28 @@ extension NumberParsing on String {
 ```
 <div class="prettify-filename">lib/string_apis.dart</div>
 
-To create a local extension that's visible only in
-the library where it's declared,
-either omit the extension name or give it a name
-that starts with an underscore (`_`).
-
-The members of the extension can be methods, getters, setters, operators.
+The members of an extension can be methods, getters, setters, or operators.
 Extensions can also have static fields and static helper methods.
 
+### Unnamed extensions
+
+You can optionally omit the name when declaring an extension.
+An unnamed extension is visible only
+in the library where it's declared.
+You also can't apply it explicitly
+to resolve [API conflicts](#api-conflicts).
+
+<?code-excerpt "extension_methods/lib/string_extensions/string_apis_unnamed.dart (unnamed)"?>
+```dart
+extension on String {
+  bool get isBlank => trim().isEmpty;
+}
+```
+
+Note that without a name,
+an extension's static members can only be invoked
+within the extension declaration.
```

Comment:
**nit:** Starting a sentence with note should either not be done or included in a note callout. If this doesn't rise to that occasion, exclude it.
```suggestion
Without a name, you can invoke an extension's
static members only within the extension declaration.
```

**question:** Is this not redundant with what's written in [lines 221 to 225](https://github.com/dart-lang/site-www/pull/4616/files#diff-3d07c88e0a0b6c36d93bad9dccd5211a89f70f70f94bedd205a61413a6b932d1R221-R225)?

Reply:
As for its relation to L221-225, it's a different class of limitation that is less likely to be come across.